lib/internal/wrap_js_stream.js

```diff
@@ -60,7 +66,10 @@ class JSStreamWrap extends Socket {
 
     super({ handle, manualStart: true });
     this.stream = stream;
-    this._list = null;
+    this[kCurrentWriteRequest] = null;
+    this[kCurrentShutdownRequest] = null;
+
+    // Start reading
```

Comment:
Dot at end.  Also, somewhat superfluous comment.

Reply:
I don’t think it’s superfluous – I could imagine there’s quite a few who wouldn’t know why you’d ever want to call `.read(0)`.